simapp/go.mod

```diff
@@ -32,13 +32,14 @@ require (
 	google.golang.org/protobuf v1.31.0
 )
 
+require cosmossdk.io/collections v0.3.0
```

Comment:
This should be in the same require group.

Reply:
Revert as unnecessary here. Ran go mo tidy in https://github.com/cosmos/cosmos-sdk/pull/17055